### PR TITLE
Implement CC callback

### PR DESCRIPTION
**Pull request by hakiemaul**

### CODE REVIEW

File: xendit.php [hunk 1]
```diff
@@ -349,8 +349,7 @@ function plgVmOnPaymentResponseReceived(&$html) {
 	 * Return:
 	 * @link: index.php?option=com_virtuemart&view=pluginresponse&task=pluginnotification&tmpl=component&xendit_mode=[xendit_mode]
```
Comment (candrasaputra) [inline xendit.php]:
```suggestion
	 * @link: index.php?option=com_virtuemart&view=pluginresponse&task=pluginnotification&tmpl=component&xendit_mode=xendit_callback
```

File: xendit.php [hunk 2]
```diff
@@ -349,8 +349,7 @@ function plgVmOnPaymentResponseReceived(&$html) {
...
 	 * xendit_mode:
-	 * 		- xendit_invoice_callback
-	 * 		- xendit_cc_callback (not support yet)
+	 * 		- xendit_callback -> support all payment method
```
Comment (candrasaputra) [inline xendit.php]:
```suggestion
	 * 
```

File: xendit.php (hunk 1 above)
Comment (candrasaputra) [reply xendit.php]: i think if we didnot have multiple xendit_mode, just pust xendit_callback here

File: xendit/lib/xendit_api.php [hunk 3]
```diff
@@ -58,6 +68,7 @@ function getInvoice($invoice_id='') {
         return $json_response;
     }
 
+    /* -Unused-
     function getInvoiceSettings() {
```
Comment (IreneGohtami) [inline xendit/lib/xendit_api.php]: function ini kalo g di pake, hapus aja

**Comment by IreneGohtami** [comment]
Buat xml file nya, tolong sekalian update nama plugin: `<name>VM Payment - Xendit Payment</name>`
Trus yg GPL license itu aku kurang ngerti apa yg harus di update. Haha.

File: xendit/lib/xendit_api.php (hunk 3 above)
Comment (hakiemaul) [reply xendit/lib/xendit_api.php]: ok

**Pull request merged**